Dockerfile

```diff
@@ -103,6 +104,12 @@ RUN cp /home/software/docker/machines.xml /home/software/catkin_ws/src/00-infras
 RUN /bin/bash -c "cd /home/software/ && source /opt/ros/kinetic/setup.bash && catkin_make -C catkin_ws/"
 RUN echo "source /home/software/docker/env.sh" >> ~/.bashrc
 
+# No point in doing this if container is going to be extended
+# RUN rm -rf /var/lib/apt/lists/*
```

Comment:
Docker layers are always additive, so when you `rm` something in a new layer, it is not really removed, the image size will only grow. So you should always run `rm -rf /var/lib/apt/lists/*` after doing an `apt-get update`. @rusi taught me this.